cpp/demo/poisson/poisson.py

```diff
@@ -21,7 +23,8 @@
 # (:math:`u` and :math:`v`) and the coefficient functions (:math:`f` and
 # :math:`g`)::
 
-coord_element = VectorElement("Lagrange", triangle, 1)
+coord_element = create_vector_element(
+    "Lagrange", "triangle", 1, lagrange_variant=basix.LagrangeVariant.equispaced)
```

Comment:
Could we have a default argument for the variant? Seems heavy to require a variant for a P1 element.

Reply:
There was a mismatch between the defaults Basix and DOLFINx were using which was causing errors. I've fixed this and removed these variants for meshes with degree < 3